### DISCUSSION TITLE
uninstalling copyQ

### DISCUSSION THREAD

**Discussion by Sairesx** [hluk/CopyQ#2215] [Q&A]
If I uninstall copyQ from my computer, does all snippets from clipboard history will be gone? (I want that)

I'm on linux, downloaded the app via flatpak (idk if that matters)

**Comment by hluk** [comment]
Use the following command to uninstall the Flatpak application and all its data including configuration and log:

```
flatpak uninstall --delete-data com.github.hluk.copyq
```

Note: CopyQ can list the configuration path (and other paths) with `flatpak run com.github.hluk.copyq info`. On my system the app data are stored in `~/.var/app/com.github.hluk.copyq/`.
